src/main.adoc

```diff
@@ -2,9 +2,13 @@
 The PartiQL Specification Committee
```

Comment:
Is it too involved to add a link to the latest released version (as adoc) at the top? if yes, maybe we can add a link to the list of all releases?

https://github.com/partiql/partiql-lang/releases

Reply:
Added!

Reply:
Thanks, and thank you for porting the previous releases!